Assets/Scripts/StaminaBar.cs

```diff
@@ -24,20 +27,13 @@ public class StaminaBar : MonoBehaviour {
 	}
 
     // Move is called each step of the players movement
-    // Returns state of stamina bar (crash or not);
+    // Returns if player can move;
     public bool move()
     {
-        Debug.Log(stamina);

```

Comment:
Please remove the debug messages before making a pull request. Its okay for play testing but its going to clog the console for everyone else